### PR TITLE
Migrate to Provisioning API nested root

### PR DESCRIPTION
**Pull request by ezr-ondrej**
Follow-up for https://github.com/RedHatInsights/image-builder-frontend/pull/1299
This updates the OpenAPI spec to the new version with the nested root.

### CODE REVIEW

File: src/store/helpers.ts [hunk 1]
```diff
@@ -1,5 +1,12 @@
...
+import type { V1ListSourceResponse } from './provisioningApi';
+
+export type OldSourceResponse = {
+  id?: string;
+  name?: string;
+  source_type_id?: string;
+  uid?: string;
+};
 
 export const extractProvisioningList = (
```
Comment (ezr-ondrej) [inline src/store/helpers.ts:10]: This is still keeping the backwards compatibility, but we can just wait with merging this a bit (few prod pushes) and then drop the backwards compatibility completely.

**Comment by lavocatt** [comment]
Hey @ezr-ondrej, this PR now conflicts with main, can you give it a rebase on your local copy please ?

**Review by lavocatt — CHANGES_REQUESTED** [review]
Hey @ezr-ondrej, this PR now conflicts with main, can you give it a rebase on your local copy please ?

File: src/store/helpers.ts (hunk 1 above)
Comment (ezr-ondrej) [reply src/store/helpers.ts:10]: It's been a while and everything seems to work fine, dropping the backwards compatibility :)

**Comment by ezr-ondrej** [comment]
> Hey @ezr-ondrej, this PR now conflicts with main, can you give it a rebase on your local copy please ?

Done and also dropped the backwards compatibility as it's already stable for a while in all environments :)

**Review by lavocatt — APPROVED** [review]
LGTM!

**Pull request merged**